vtk-cdat/build.sh

```diff
@@ -107,34 +108,53 @@ COMMON_ARGS="-DCMAKE_C_COMPILER=$CC \
         -DVTK_Group_StandAlone:BOOL=OFF \
         -DVTK_LEGACY_SILENT:BOOL=ON"
 
+if [ ${OSNAME} == Linux ]; then
+    LIBEXT="so"
+elif [ ${OSNAME} == Darwin ]; then
+    LIBEXT="dylib"
+fi
 
-if [ `uname` == Linux ]; then
+VTK_ARGS="\
+    -DCMAKE_OSX_DEPLOYMENT_TARGET=10.12 \
```

Comment:
Actually this was already in the recipe and I merely moved it to the `VTK_ARGS` variable.

@doutriaux1 Any reason why we target the latest macOS?